app/script/cryptography/CryptographyRepository.js

```diff
@@ -50,15 +50,49 @@ z.cryptography.CryptographyRepository = class CryptographyRepository {
   }
 
   /**
+   * Initializes the repository by loading an existing cryptobox.
+   * @param {Object} database - Database object
+   * @returns {Promise} Resolves after initialization
+   */
+  create_cryptobox(database) {
+    return this._init(database)
+      .then(() => this.cryptobox.create());
+  }
+
+  /**
+   * Initializes the repository by creating a new cryptobox.
+   * @param {Object} database - Database object
+   * @returns {Promise} Resolves after initialization
+   */
+  load_cryptobox(database) {
+    return this._init(database)
+      .then(() => this.cryptobox.load());
+  }
+
+  reset_cryptobox(client_et) {
+    const delete_everything = client_et ? client_et.is_temporary() : false;
+    const delete_promise = delete_everything ? this.storage_repository.delete_everything() : this.storage_repository.delete_cryptography();
+
+    return delete_promise
+      .catch((database_error) => {
+        this.logger.error(`Deleting crypto database after failed client validation unsuccessful: ${database_error.message}`, database_error);
```

Comment:
What's the "crypto database"? Maybe better write "Deleting cryptography-related records..."